test/tests/functional/pbs_resv_start_dur_end.py

```diff
@@ -90,3 +89,46 @@ def test_duration_end_resv_fail(self):
         self.assertTrue(rid is None,
                         'Reservation Submit succeeded' +
                         'when it should have failed')
+
+    def test_start_dur_end_resv_fail(self):
+        """
+        Test to submit a job with a start, end, and duration
+        where start + duration != end.
+        """
+        # create reservation to start in 10 seconds
+        # to end in 30 seconds
+        # and to last 10 seconds
```

Comment:
This comment isn't really necessary.  The code is pretty clear that you are submitting.  If you want to have a comment, explain why you are submitting this reservation.  You've explained why in the doc string right above, so I don't really think the comment is necessary.